### ISSUE TITLE
Tablist container on Pages page is empty

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6788]
### Descriptive summary

The tablist container on the Pages page is empty and does not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. As an admin user, go to https://dev.nurax.samvera.org/pages/edit?locale=en and run the Site Improve browser extension

![pages_empty_container](https://github.com/samvera/hyrax/assets/32644254/e705fcf6-f85b-42f6-8a57-422b61538a6d)

**Comment by aahurford** [comment]
tested and confirmed on pg

**Comment by christhepianist** [comment]
Tested and confirmed that this is no longer an issue.
